internal/terminal/table.go

```diff
@@ -0,0 +1,151 @@
+package terminal
+
+import (
+	"fmt"
+	"github.com/fatih/color"
+	"regexp"
+	"strings"
+)
+
+// table
+// headers is an ordered list of strings to specify the order of printing
+// data is a map of a string to another string to print, where the key should be a value in headers
+//	if not, we will not print
+// maxHeaderWidth is to determine the column width based on the max length of the values and header
+type table struct {
+	headers        []string
+	data           []map[string]string
+	maxHeaderWidth map[string]int
+}
+
+// columnBreakWidth is to set a standard for the break between columns for the table
+const (
+	 columnBreakWidth = 2
+)
+
+func newTable(headers []string, values []map[string]interface{}) *table {
+	if headers == nil || len(headers) == 0 ||
+		values == nil || len(values) == 0 {
+		return nil
+	}
+
+	maxHeaderWidth := make(map[string]int)
+	for _, h := range headers {
+		temp := len(h)
+		if temp > maxHeaderWidth[h] {
+			maxHeaderWidth[h] = temp
+		}
+	}
+
+	var stringValues []map[string]string
+	for _, v := range values {
+		tempVal := make(map[string]string)
+		for k1, v1 := range v {
+			// convert the interface values into strings for printing
+			var val string
+			switch v := v1.(type) {
+			case string:
+				spaceRegex := regexp.MustCompile(`\s+`)
+				val = spaceRegex.ReplaceAllString(v, " ")
+			case float32, float64:
```

Comment:
lets remove this `case` until we need it (feel free to let me know I'm missing something and we need it now)

Reply:
the float gets cut off at a certain number, so i make it formatted by %g (which handles long floats and defaults to %f if it's short)

Reply:
neat!  i definitely did not know about the `%g` formatter... i guess my question here is: what `float`s are already trying to print here?  if this was just for future-proofing, then I'll still suggest we remove this `case` until the need/requirement for it shows up